### ISSUE TITLE
Fatal error: undefined is not a function

### ISSUE THREAD

**Issue by felicienfrancois** [rakuten-frontend/grunt-appdmg#4]
With node v0.12.7 (latest available with brew), I get a fatal error:

```
Running "appdmg:target" (appdmg) task
Fatal error: undefined is not a function
TypeError: undefined is not a function
    at EventEmitter.<anonymous> (/Users/felicienfrancois/git/TweakStyle/node_modules/grunt-appdmg/tasks/appdmg.js:39:41)
    at EventEmitter.emit (events.js:107:17)
    at Pipeline._progress (/Users/felicienfrancois/git/TweakStyle/node_modules/grunt-appdmg/node_modules/appdmg/lib/pipeline.js:20:11)
    at Pipeline._runStep (/Users/felicienfrancois/git/TweakStyle/node_modules/grunt-appdmg/node_modules/appdmg/lib/pipeline.js:45:8)
    at Pipeline._run (/Users/felicienfrancois/git/TweakStyle/node_modules/grunt-appdmg/node_modules/appdmg/lib/pipeline.js:97:8)
    at process._tickCallback (node.js:355:11)
    at Function.Module.runMain (module.js:503:11)
    at startup (node.js:129:16)
    at node.js:814:3
```

This is due to the use of String.repeat in tasks/appdmg.js:39 which is not defined in nodejs 0.12.7

**Comment by felicienfrancois** [comment]
Could be fixed with either https://github.com/paulmillr/es6-shim or https://github.com/mathiasbynens/String.prototype.repeat polyfills

**Comment by felicienfrancois** [comment]
Or even cleaner, Use [grunt.util.repeat](http://gruntjs.com/api/grunt.util#grunt.util.repeat)
I just made a pull request https://github.com/rakuten-frontend/grunt-appdmg/pull/5

**Comment by ducky427** [comment]
I've also experienced the same issue.

**Comment by htanjo** [comment]
Fixed by #5.
And I have released `grunt-appdmg v0.3.2`.

Thanks!
